config/software/openssl.rb

```diff
@@ -23,14 +23,14 @@
 dependency "makedepend"
 
 
-default_version "1.0.1t"
+default_version "1.0.2h"
 source url: "ftp://ftp.openssl.org/source/#{name}-#{version}.tar.gz", # FIXME: use https as soon as we upgrade to debian 7
```

Comment:
unrelated but let's do it anyway: we don't support debian 6 anymore and our debian docker images are now based on debian 7, so let's use `https` here! :)